moviepy/Clip.py

```diff
@@ -429,15 +427,13 @@ def cutout(self, start_time, end_time):
         """
 
         new_clip = self.time_transform(
-            lambda t: t + (t >= start_time) * (end_time - start_time)
+            lambda t: t + (t >= start_time) * (end_time - start_time),
+            apply_to=["audio", "video"],
```

Comment:
```suggestion
            apply_to=["mask", "audio"],
```
`"video"` does nothing here (because no clips have a `.video` attribute.

Reply:
Ups :sweat_smile: yes